frontend/src/App.js

```diff
@@ -36,7 +36,7 @@ function App() {
     <>
       <Route path="/profile" element={<ProfilePage />} />
       <Route path="/courses" element={<CourseIndexPage />} />
-      <Route path="/schools" element={<SchoolIndexPage />} />
+      <Route path="/courses/create" element={<CoursesCreatePage />} />
     </>
```

Comment:
It looks llike you are removing the schools index and replacing it with /courses/create.

Are you really sure that's what you want to do here?